code/renderers/react/src/docs/jsxDecorator.tsx

```diff
@@ -124,16 +125,28 @@ export const renderJsx = (code: React.ReactElement, options?: JSXOptions) => {
   } else {
     displayNameDefaults = {
       // To get exotic component names resolving properly
-      displayName: (el: any): string =>
-        el.type.displayName || typeof el.type === 'symbol'
-          ? getReactSymbolName(el.type)
-          : null ||
-            getDocgenSection(el.type, 'displayName') ||
-            (el.type.name !== '_default' ? el.type.name : null) ||
-            (typeof el.type === 'function' ? 'No Display Name' : null) ||
-            (isForwardRef(el.type) ? el.type.render.name : null) ||
-            (isMemo(el.type) ? el.type.type.name : null) ||
-            el.type,
+      displayName: (el: any): string => {
+        if (el.type.displayName) {
+          return el.type.displayName;
+        } else if (getDocgenSection(el.type, 'displayName')) {
```

Comment:
I moved this up because I am assuming that docgenInfo should take precedence over an element's symbol name. 

Before, if you had a forwardRef element AND docgenInfo with a displayName of "Button", it would say <React.ForwardRef> instead of <Button>.

Now it will say <Button>.